api/tests/Api/SnapshotTests/__snapshots__/ResponseSnapshotTest__testGetCollectionMatchesStructure with data set camp_collaborations__1.json

```diff
@@ -869,6 +1240,9 @@
                             },
                             "self": {
                                 "href": "escaped_value"
+                            },
+                            "sharedBy": {
```

Comment:
Here too... is this correct

Reply:
Due to all the escaping I can't tell which camp this is, but assuming this camp has a sharedBy set then that snapshot looks correct to me.